### PR TITLE
feat(INVT-CPC-959): add react as the new front end

### PR DESCRIPTION
**Pull request by nic5694**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CPC-959
## Context:
Created new front-end in react
## Changes
- Changed the front end to react
- Added the project and integrated react router
## Before and After UI (Required for UI-impacting PRs)
image
## Dev notes (Optional)
- More of the front end will be added over time....
- Migration to react starts now!

### CODE REVIEW

**Review by nic5694 — COMMENTED** [review]
this looks rly good!

File: api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/BFFApiGatewayController.java [hunk 1]
```diff
@@ -46,6 +46,7 @@
 @Slf4j
 @RequestMapping("/api/gateway")
 @Validated
+@CrossOrigin(origins = "http://localhost:3000")
```
Comment (Sprytte) [inline api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/BFFApiGatewayController.java:49]: good job, on tinking about the cross origin

**Review by Sprytte — APPROVED** [review]
waw thank you so much, you are doing god's work!

**Review by Sprytte — APPROVED** [review]
good job!

**Review by Eli293 — APPROVED** [review]
Looks good to me!

**Comment by cgerard321** [comment]
I'm so excited about this!

**Pull request merged**